### ISSUE TITLE
[Feature Request] Dynamic Theme changing based on os-parameters

### ISSUE THREAD

**Issue by zDEFz** [revoltchat/revolt#486]
### What do you want to see?

# Dynamic Theme changing based on os-parameters
- https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
- For Accessibility, find a way to also detect high-contrast-white and high-contrast-black themes.

### PWA

- [ ] Yes, this feature request is specific to the PWA.
